servers/republik/graphql/resolvers/Comment.js

```diff
@@ -156,25 +156,17 @@ module.exports = {
     if (comment.displayAuthor) {
       return comment.displayAuthor
     }
+    console.log('load displayUser')
 
-    const commenter = _commenter ||
-      transformUser(
-        await pgdb.public.users.findOne({
-          id: comment.userId
-        })
-      )
-    const commenterPreferences = _commenterPreferences ||
-      await pgdb.public.discussionPreferences.findOne({
-        userId: commenter.id,
-        discussionId: comment.discussionId
-      })
-    const discussion = _discussion ||
-      await pgdb.public.discussions.findOne({
-        id: comment.discussionId
-      })
-    const credential = commenterPreferences && commenterPreferences.credentialId
-      ? _credential || await pgdb.public.credentials.findOne({ id: commenterPreferences.credentialId })
-      : null
+    const { loaders } = context
+    const commenter = await loaders.User.byId.load(comment.userId)
+    const commenterPreferences = await loaders.Discussion.Commenter.discussionPreferences.load({
```

Comment:
this could run in parallel:
```js
const [commenter, commenterPreferences] = await Promise.all([
  loaders.User.byId.load(comment.userId),
  loaders.Discussion.Commenter.discussionPreferences.load({
    userId: comment.userId,
    discussionId: comment.discussionId
  })
])
```